avogadro/quantumio/orca.cpp

```diff
@@ -374,6 +388,10 @@ void ORCAOutput::processLine(std::istream& in, GaussianSet* basis)
           // the first entry might be 5 or 6 because of removed rotations /
           // translations
           int index = Core::lexicalCast<int>(list[0]);
+          // invalid index
```

Comment:
Shouldn't we just declare (index) as `std::size_t` above? In that case, `index > 0` below can be dropped too.

Reply:
Yes.